apps/climatemappedafrica/src/components/HowItWorks/index.js

```diff
@@ -0,0 +1,200 @@
+import { RichText } from "@commons-ui/payload";
+import { useMediaQuery, Box, Button, Grid, Typography } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Player from "@/climatemappedafrica/components/HowItWorks/Player";
+import useStyles from "@/climatemappedafrica/components/HowItWorks/useStyles";
+import Section from "@/climatemappedafrica/components/Section";
+
+function HowItWorks({
+  title,
+  description,
+  link,
+  video,
+  backgroundImage,
+  foregroundImage,
+  ...props
+}) {
+  const classes = useStyles(props);
+
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+
+  return (
+    <Box
+      sx={{
+        height: theme.typography.pxToRem(820),
+        position: "relative",
+        [theme.breakpoints.up("md")]: {
+          height: theme.typography.pxToRem(618),
+        },
+        [theme.breakpoints.up("lg")]: {
+          height: theme.typography.pxToRem(728),
+        },
```

Comment:
You don't need `theme.breakpoints`:
```suggestion
        height: {  xs: theme.typography.pxToRem(820), md: theme.typography.pxToRem(618), lg: theme.typography.pxToRem(728)},
        position: "relative",
```

or much simpler:
```jsx
        height: {  xs: "820px", md: "618px", lg: "728px" },
        position: "relative",
```